server/config.go

```diff
@@ -28,10 +28,8 @@ type Profile struct {
 	BackendHost string
```

Comment:
`BackendHost` seems irrelevant too.
We are starting the echo server using `s.e.Start(fmt.Sprintf(":%d", s.profile.BackendPort))`, IMO which basically says "listen on any available host on port BackendPort".

Reply:
And if we are really listening to `localhost`, it is actually not desired. The external gateway cannot contact the Bytebase server because they are not on the same host.

Reply:
How about make `flags.host` default to 0.0.0.0?

Reply:
Done

Reply:
Done